css/src/components/pagination.scss

```diff
@@ -0,0 +1,195 @@
+$font-size-lg: $font-size-6 !default;
+$font-size-sm: $font-size-8 !default;
+
+$pagination-color: $text !default;
+$pagination-border-color: $border !default;
+$pagination-margin: 0.25rem !default;
+
+$pagination-hover-color: $primary-hover !default;
+$pagination-hover-border-color: $primary-hover !default;
+
+$pagination-focus-color: $primary-active !default;
+$pagination-focus-border-color: $primary-active !default;
+
+$pagination-active-color: $primary-active !default;
+$pagination-active-border-color: $primary-active !default;
+
+$pagination-disabled-color: $secondary !default;
+$pagination-disabled-background-color: $secondary-background !default;
+$pagination-disabled-border-color: $secondary-background !default;
+
+$pagination-current-color: $text-invert !default;
+$pagination-current-background-color: $primary !default;
+$pagination-current-background-color-hover: $primary-hover !default;
+$pagination-current-border-color: $text-invert !default;
+
+$pagination-ellipsis-color: $secondary-background !default;
+
+$pagination-shadow-inset: inset 0 1px 2px $box-shadow-color-light;
+
+.pagination {
+	flex-wrap: wrap;
+	margin: -$pagination-margin;
+}
+
+.pagination,
+.pagination-list {
+	display: flex;
+	flex-grow: 1;
+	flex-wrap: wrap;
+	align-items: center;
+	justify-content: center;
+	margin: 0;
+	text-align: center;
+
+	> li {
+		margin: $pagination-margin;
+		list-style: none !important; //specificity battle
+	}
+}
+
+.pagination-link {
+	&.is-current {
+		border-color: $pagination-current-border-color;
+		background-color: $pagination-current-background-color;
+		color: $pagination-current-color;
+
+		&::hover {
+			background-color: $pagination-current-background-color-hover;
+			color: $pagination-current-color;
+		}
+	}
+
+	@include forced-colors {
+		&::after {
+			display: block;
+			position: absolute;
+			width: 0.375em;
+			height: 0.375em;
+			border: 1px solid LinkText;
+			border-radius: $border-radius-rounded;
+			background-color: LinkText;
+			content: '';
+			inset-block-start: 0.25em;
+			inset-inline-end: 0.25em;
+			forced-color-adjust: none;
+		}
+	}
+}
+
+.pagination-ellipsis {
+	color: $pagination-ellipsis-color;
+	pointer-events: none;
+}
+
+.pagination-previous,
+.pagination-next {
+	padding-right: 0.75em;
+	padding-left: 0.75em;
+	white-space: nowrap;
+}
+
+.pagination-previous,
+.pagination-next,
+.pagination-link,
+.pagination-ellipsis {
+	@include control;
+	@include unselectable;
+	@include transparent-effects;
+
+	flex-grow: 0;
+	flex-shrink: 1;
+	justify-content: center;
+	padding-right: 0.5em;
+	padding-left: 0.5em;
```

Comment:
`padding-inline: .5em`

Reply:
(replaces both padding-left and padding-right)

Reply:
updated